### ISSUE TITLE
~Priority, Submit Free Listing Fields to Be Added~See comment below....

### ISSUE THREAD

**Issue by urentmellc** [pixelphpdev/urentme#36]
Add: full address, Item Name, Make, Model, Year, Weekly Rate, Monthly. All fields must be filled out in total.

**Comment by Jayantapixel** [comment]
Work is in progress... We will test and release soon.

**Comment by pxlsz** [comment]
Add listing form updated as per your requirement. Please check.

**Comment by urentmellc** [comment]
was this tested on your end? With a new listing?

**Comment by Jayantapixel** [comment]
Yes we have tested using dummy data and its working fine. We have deleted that after testing. You can check at your end using a proper data.
